app/models/Campaign.js

```diff
@@ -34,8 +34,12 @@ const campaignSchema = new mongoose.Schema({
   msg_menu_signedup_external: String,
   msg_menu_signedup_gambit: String,
   msg_no_photo_sent: String,
-  msg_relative_reminder_signup: String,
-  msg_relative_reminder_reportback: String,
+
+  // Exposed messages.
+  messages: {
```

Comment:
We've discussed the wording issue in the team and have agreed to name first `relative_to_signup` and second `relative_to_reportback`. Let me know if it you're ok with it @aaronschachter.